--- dmapi-2.0.8/m4/manual_format.m4.orig	2003-06-25 03:51:30.000000000 +0200
+++ dmapi-2.0.8/m4/manual_format.m4	2003-08-08 21:25:15.000000000 +0200
@@ -6,12 +6,5 @@
 # 
 AC_DEFUN([AC_MANUAL_FORMAT],
   [ have_zipped_manpages=false
-    for d in ${prefix}/share/man ${prefix}/man ; do
-        if test -f $d/man1/man.1.gz
-        then
-            have_zipped_manpages=true
-            break
-        fi
-    done
     AC_SUBST(have_zipped_manpages)
   ])
--- a/libdm/dm_handle2path.c
+++ b/libdm/dm_handle2path.c
@@ -20,6 +20,9 @@
 #include <dmapi_kern.h>
 #include "dmapi_lib.h"
 
+#include <fcntl.h> /* open */
+#include <unistd.h> /* close */
+#include <string.h> /* strlen */
 #include <mntent.h>
 #include <dirent.h>
 #ifdef linux
